qiskit_ibm_runtime/transpiler/plugin.py

```diff
@@ -96,3 +97,41 @@ def pass_manager(
             plugin_passes += [ConvertConditionsToIfOps()]
 
         return PassManager(plugin_passes) + translator_pm
+
+
+class IBMFractionalTranslationPlugin(PassManagerStagePlugin):
+    """A translation stage plugin for targeting Qiskit circuits
+    to IBM Quantum systems with fractional gate support.
+
+    Currently coexistence of fractional gate operations and
+    dynamic circuits is not assumed.
+    """
+
+    def pass_manager(
+        self,
+        pass_manager_config: PassManagerConfig,
+        optimization_level: Optional[int] = None,
+    ) -> PassManager:
+        """Build IBMTranslationPlugin PassManager."""
+
+        translator_pm = common.generate_translation_passmanager(
+            target=pass_manager_config.target,
+            basis_gates=pass_manager_config.basis_gates,
+            approximation_degree=pass_manager_config.approximation_degree,
+            coupling_map=pass_manager_config.coupling_map,
+            backend_props=pass_manager_config.backend_properties,
+            unitary_synthesis_method=pass_manager_config.unitary_synthesis_method,
+            unitary_synthesis_plugin_config=pass_manager_config.unitary_synthesis_plugin_config,
+            hls_config=pass_manager_config.hls_config,
+        )
```

Comment:
We have recently added a new argument ``qubits_initially_zero`` to ``generate_pass_manager``, indicating whether circuit starts in the ``|0>``-state. You may want to propagate its value along with the other arguments. (BTW, I am not sure that I have indented the additional line correctly).

```suggestion
            qubits_initially_zero=pass_manager_config.qubits_initially_zero,
        )
```

Reply:
Thanks. I prefer not in this PR (out of scope), so I've opened #2068.